CMakeLists.txt

```diff
@@ -80,8 +81,12 @@ include_directories(${FFTW3_INCLUDES})
 #Process GROMACS settings
 ########################################################################
 
-find_package(GROMACS REQUIRED)
-include_directories(${GROMACS_INCLUDES})
+find_package(GROMACS CONFIG)
```

Comment:
This should be already in `if (ENABLE_GROMACS)` block, otherwise end-user will see beautiful CMake warning;